react - can't build react library when pnpm is used

## Current Behavior

When workspace is managed  with `pnpm`, `build` command will fail due to missing dependency. Works fine with `npm` and `yarn`.

## Expected Behavior

Should work :)

## Steps to Reproduce

```
pnpx create-nx-workspacereact-buildable-bug
cd react-buildable-bug
pnpm install
pnpm install @nrwl/react
pnpm run nx g @nrwl/react:library ui -- --name=ui --style=none --buildable
```


### Failure Logs

```
> react-buildable-bug@0.0.0 nx ***\react-buildable-bug
> nx "run" "ui:build"

> nx run ui:build
Bundling ui...
Error during bundle: Error: Cannot find module 'babel-plugin-transform-async-to-promises' from '***\react-buildable-bug\libs\ui\src'
Bundle failed: ui
Error during bundle: Error: Cannot find module '@nrwl/web/babel' from '***\react-buildable-bug'
Bundle failed: ui
ERROR  Command failed with exit code 1.
```


### Environment

```
 Node : 14.15.0
  OS   : win32 x64
  yarn : 1.22.4

  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.2
  @nrwl/cypress : 11.0.15
  @nrwl/devkit : 11.0.15
  @nrwl/eslint-plugin-nx : 11.0.2
  @nrwl/express : Not Found
  @nrwl/jest : 11.0.15
  @nrwl/linter : 11.0.15
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 11.0.15
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.2
  @nrwl/web : 11.0.15
  @nrwl/workspace : 11.0.2
  typescript : 4.0.5
```

Thank you,

Rafał

--- a/packages/web/src/builders/package/package.impl.ts
+++ b/packages/web/src/builders/package/package.impl.ts
@@ -222,7 +222,7 @@ export function createRollupOptions(
         plugins: [
           config.format === 'esm'
             ? undefined
-            : 'babel-plugin-transform-async-to-promises',
+            : require.resolve('babel-plugin-transform-async-to-promises'),
         ].filter(Boolean),
       }),
       commonjs(),

